cnxarchive/views.py

```diff
@@ -981,3 +1016,47 @@ def robots(request):
     resp.expires = html_date(exp_time)
     resp.body = robots_dot_txt.to_string()
     return resp
+
+@view_config(route_name='recent', request_method='GET',
+             accept="application/json",
+             renderer='templates/recent.rss') #renderer: cnxarchive.views:templates/recent.html?? also what should the permission actuall be??
+def recent(request):
+    # setting the query variables
+    query_vars = get_query_vars(request.query_string)
```

Comment:
It's not necessary to parse this yourself, though good job in doing so. See the [pyramid.request.Request.GET documentation](http://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.GET)

An example usage of this might be `num_entries = request.GET.get('number', 10)`